external-crates/move/move-compiler/src/parser/syntax.rs

```diff
@@ -517,13 +517,15 @@ fn parse_visibility(context: &mut Context) -> Result<Visibility, Box<Diagnostic>
     let loc = make_loc(context.tokens.file_hash(), start_loc, end_loc);
     Ok(match sub_public_vis {
         None => Visibility::Public(loc),
-        Some(Tok::Script) => Visibility::Script(loc),
-        Some(Tok::Friend) => Visibility::Friend(loc),
+        Some((Tok::Script, _)) => Visibility::Script(loc),
+        Some((Tok::Friend, _)) => Visibility::Friend(loc),
+        Some((Tok::Identifier, Visibility::PACKAGE_IDENT)) => Visibility::Package(loc),
         _ => {
             let msg = format!(
-                "Invalid visibility modifier. Consider removing it or using '{}' or '{}'",
-                Visibility::PUBLIC,
-                Visibility::FRIEND
+                "Invalid visibility modifier. Consider removing it or using '{}', '{}' or '{}'",
+                Visibility::FRIEND,
+                Visibility::PACKAGE,
+                Visibility::SCRIPT,
```

Comment:
Nit: I believe we probably want to deprecate `public(script)` in the 2024 edition (please correct me if I'm wrong though @tnowacki) so probably best to not reccomend it as a possible visibility.